fastlane/Fastfile

```diff
@@ -0,0 +1,161 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+require("json")
+default_platform(:android)
+
+platform :android do
+  lane :testConnectionToGooglePlay do
+    validate_play_store_json_key(
+      json_key_data: ENV["GOOGLE_PLAY_PRIVATE_KEY"]
+    )
+  end
+
+  desc "Runs all the tests"
+  lane :test do
+    gradle(
+      task: "test",
+      build_type: "Release",
+    )
+  end
+
+  desc "Creates a Sentry release for this version and associates Git commits"
+  lane :createSentryRelease do
+    sentry_check_cli_installed()
+    #     gradle(task: "clean assembleRelease")
+    version_name = android_get_version_name(
+        gradle_file: "app/build.gradle.kts"
+    )
+    version_code = android_get_version_code(
+        gradle_file: "app/build.gradle.kts"
+    )
+    sentry_create_release(
+        auth_token: ENV["SENTRY_AUTH_TOKEN"],
+        org_slug: "robojackets",
+        project_slug: "apiary-android",
+        version: "#{version_name}+#{version_code}",
+        app_identifier: "org.robojackets.apiary",
+        finalize: true
+    )
+#     sentry_set_commits(
+#         auth_token: "",
+#         org_slug: "robojackets",
+#         project_slug: "apiary-android",
+#         version: "#{version_name}+#{version_code}",
+#         app_identifier: "org.robojackets.apiary",
+#         auto: true
+#     )
```

Comment:
Why is this commented out?

Reply:
Sentry was just an extra complication on top of the other stuff initially, so I decided to punt all Sentry release automation to CI round 2